mlp/training/mlp_model.py

```diff
@@ -64,7 +66,7 @@ def generate_default_model(self, num_labels):
             nout=250,
             init=init_norm,
             bias=bias_init,
-            activation=Rectlin()))
+            activation=Tanh()))
```

Comment:
Yes—insofar as I wanted to test that we can load it and use it in iOS. It happens to perform slightly better for the core dataset, too.